composer.json

```diff
@@ -10,10 +10,10 @@
     ],
     "require": {
         "guzzle/http": "~3.7",
-        "jms/serializer": "0.12.*",
+        "jms/serializer": "~1.1",
         "psr/log": "~1.0",
-        "symfony/console": "~2.3",
-        "symfony/expression-language": "~2.4"
+        "symfony/console": "~2.8",
+        "symfony/expression-language": "~2.8"
```

Comment:
You're right... I did not think about it. 
I followed this post : http://blog.insight.sensiolabs.com/2014/02/12/jenkins-integration.html and I downloaded the "phar" archive. I suppose that the versions of the included libraries are a bit old for PHP7 ?